examples/timeline/markers/customTimeMarkersEditable.html

```diff
@@ -30,8 +30,8 @@
   };
   var timeline = new vis.Timeline(container, items, options);
 
-  var id1 = new Date().getTime();
-  var id2 = new Date().getTime();
+  var id1 = '93b8a81c-c1fd-4800-9cdd-ee8d0ac445b0';
+  var id2 = '19ddf917-5434-4bb8-a8c7-24ca09eb7e09';
```

Comment:
Why a long `uuid`? Can we not simply use a simple `id1` and `id2` strings?

Reply:
I think both of id strings would not cause any problems in this case. So it seems certainly better to use  the simple ids instead of the uuids.

Reply:
Fixed in 7079116